src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -172,11 +173,10 @@ private function conditions_ok( $node_data, $element ) {
 		return $conditions_meet;
 	}
 
-	public function initialize_nodes_to_translate() {
-
-		$this->nodes_to_translate = array(
+	public static function get_nodes_to_translate() {
```

Comment:
@strategio why changing it to static? We don't use static methods here

Reply:
@andreasciamanna, the returned array do not depends on the state of the object. It's more a configuration array.

Reply:
But all classes implementing `IWPML_Page_Builders_Translatable_Nodes` use non-satic method for `initialize_nodes_to_translate`.

The interface itself does not declare the method as static.

Declaring it static only in `class-wpml-elementor-translatable-nodes.php` not only makes the implementation inconsistent but violates the interface's contract.

Reply:
@andreasciamanna, I didn't remove `initialize_nodes_to_translate`, it's still here https://github.com/OnTheGoSystems/wpml-page-builders-elementor/pull/52/files#diff-1e5046e82faeba27c319819ef7a502caR673.

Reply:
I've confused one method with the other. Sorry.